src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -47,8 +48,9 @@ public partial class ScriptEditorWindow : ModelessChildWindow
             Dynamo.Logging.Analytics.TrackScreenView("Python");
         }
 
-        internal void Initialize(Guid nodeGuid, string propName, string propValue)
+        internal void Initialize(Guid workspaceGuid, Guid nodeGuid, string propName, string propValue)
```

Comment:
this should be fine as it's an internal method.

Reply:
The entire class probably ought to be internal instead of public, as it's not useful without being able to call the `Initialize` method.

Reply:
@Dewb In this case, can you make a comment to the class, e.g. TODO: Make this class internal in Dynamo 3.0? By the time we tackle it, we would remember to do this.